app/waf/src/models/monitor.model.ts

```diff
@@ -0,0 +1,46 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
```

Comment:
Suggest to include all possible monitor properties, so that we will have all columns defined in db. It will be easier to support new types of monitor in the future.